app.js

```diff
@@ -13,14 +13,14 @@ const catchphraseInput = document.getElementById('catchphrase-input');
 const catchphraseButton = document.getElementById('catchphrase-button');
 
 // set state for how many times the user changes the head, middle, and bottom
-const timesChanged = document.getElementById('stats');
+const reportEl = document.getElementById('report');
```

Comment:
`reportEl` is`null` because the `<p>` tag in your HTML with the 'report' ID was deleted